### DISCUSSION TITLE
how to multicasting in sunshine?

### DISCUSSION THREAD

**Discussion by lasxmidnight78** [LizardByte/community#138] [Sunshine] [answered]
there is a channel option in the website，its said it can multicasting，but when i set it to 3,it doesn't work.
when i use pc streaming to pc, and then i want to use the cellphone streaming，,it just full blackscreen,or shutdown it self

**Comment by cgutman** [comment, marked as answer]
Multicasting is broken in v0.17.0 on Windows. The issue is tracked as #795
